cf-backup/files/bin/run-rack.sh

```diff
@@ -0,0 +1 @@
+rack files object upload-dir --container $CONTAINER --dir $VOLUME
```

Comment:
You might want to consider adding an option to tar/gz the dir before upload. That would make the upload a lot more efficient.